proc.c

```diff
@@ -3811,6 +3811,54 @@ proc_ruby2_keywords(VALUE procval)
  * Since +return+ and +break+ exits the block itself in lambdas,
  * lambdas cannot be orphaned.
  *
+ * == Numbered parameters
+ *
+ * To simplify writing short blocks, Ruby 2.7 have introduced numbered
```

Comment:
While it's valuable information, I don't think the version in which this feature was introduced should be such a prominent part of this section. Docs like this should be written for all posterity and the introduction date becomes less and less interesting to readers as the feature gains general recognition. I don't recall other documentation about core language features mentioning introduction dates, either.

Maybe it would be better to motivate this feature in more detail in the first sentence, something like:
"Numbered parameters are implicitly defined block parameters intended to simplify writing short blocks".

Reply:
Makes sense, applied. 
TBH, I've recently read some new language docs (was it Dart?..) and noticed features in the reference frequently have "It was introduced in version X, in order to this and that" and thought about it as kinda appealing, but probably this approach is more suiting young, quickly changed languages.